docs/provisioning.rst

```diff
@@ -24,7 +24,7 @@ We've used this AWS AMI::
 Initial Setup
 ------------------------
 
-This project uses Python 3.3.  Because Fabric does not support Python 3, you will need Fabric
+This project uses Python 3.4.  Because Fabric does not support Python 3, you will need Fabric
```

Comment:
Maybe just say ``uses Python 3`` here so it doesn't get obsolete again?

Reply:
Should I change the other '3.4' locations to '3' as well, then?

Reply:
I just suggested it here because any release of Python 3 would have the problem we're discussing. I think in other places it might be important to be more specific about what we're using?  But if not, then sure, no need to be over-specific.